tests/conftest.py

```diff
@@ -163,7 +165,15 @@ def mock_manifest_cache_buster():
 
 
 @pytest.fixture(scope="session")
-def app_config(database):
+def redis():
+    return {
+        "sessions.url": "redis://redis:0/",
+        "warehouse.xmlrpc.cache.url": "redis://redis:0/",
+    }
```

Comment:
Ah yeah, almost forgot about that. I had to change the URL for redis, but not exactly sure how I should do about that:
- Either find a way so that it's not actually required, to ease Travis configuration
- Or configure redis in Travis (well, if it's not already there)